_harp/apply/_apply.md

```diff
@@ -1,75 +1,57 @@
-
-  When we receive your Expression of Interest, we will invite you to our Coding for Everyone community on [Gitter](https://gitter.im/codingforeveryone). Please join us and introduce yourself.
-  If you do not receive an invitation within 3 days, please resubmit your Expression of Interest, checking that you have entered your GitHub username accurately.
-
-
-### STEP 3
-  Before completing the application form:
-
-  1) Create Free Code Camp Account(see #1 under COURSE PREREQUISITES)
-
-  2) Create Codewars Account(see #2 under COURSE PREREQUISITES)
-
-  3) Prepare an application video that answers these questions (if you do not want to video yourself, please do a screencast instead):
-
-   + Why do you want to join the programme?
-   + What problem would you most like to solve with technology?
-   + What do you intend to do when you finish?
-   + What really motivates you?
-
-
-  Please, Make sure to keep it to under two minutes and record your voice at an adequate sound level.
-  You can use YouTube or Vimeo or similar service to upload your video.
+<h2 align='center'>COURSE PREREQUISITES</h2>
 
-  [Submit your application here](https://goo.gl/ENPIlD).
+<h3 align='center'>CODING FOR EVERYONE</h3>
 
-  You can expect to hear from us within 14 days of the application closing date.
+Each of our campuses hosts one or more regular meetups for all people interested in learning web development. Please make every effort to attend one of our local meetups:
 
-<h2 align='center'>COURSE PREREQUISITES</h2>
-<h3> In order for your application to be considered, you should make sure you complete these three steps before the application closing date.</h3>
-We suggest you start working on Free Code Camp first, and when you feel comfortable enough with Javascript,  start solving Katas on CodeWars while finishing Free Code Camp required sections.
+London: please check [meetup.com](https://www.meetup.com/founderscoders/) for our meetup schedule.
++ Coding for Everyone: every Monday, 6:30 PM, 14 Palmers Road
++ Coding for Women: every other Tuesday, 6:30 PM, 14 Palmers Road
 
-### 1)
+Nazareth:
++ Coding for Everyone, every Monday, 6:30 PM, [Khan el-Basha building](https://waze.to/lr/hsvc451n7h)
 
-  Sign up to [Free Code Camp](http://www.freecodecamp.com/) then complete the following sections on "Front End Development" course:
-  + HTML5 & CSS
-  + Basic JavaScript
-  + Object Oriented and Functional Programming
-  + Basic Algorithm Scripting
+<a href="http://www.github.com/"
+  alt="GitHub website">
+  <img class="partners-section-image" alt="GitHub logo" src="http://www.aha.io/assets/github.7433692cabbfa132f34adb034e7909fa.png"/>
+</a>
 
-### 2)
+1. Create a GitHub account (note: you can easily use this account to join Free Code Camp and Codewars, our other prerequisites)
+2. Join our online community on [Gitter](https://gitter.im/home/explore), and actively participate in the chatroom of the course for which you are applying:
```

Comment:
Sorted now. I just had to open some new rooms first, that aren't private. Now we don't have to keep manually inviting people :tada: